src/main/kotlin/org/elm/ide/typing/ElmOnEnterIndentHandler.kt

```diff
@@ -35,7 +36,7 @@ import org.elm.lang.core.psi.elements.ElmTypeDeclaration
  * "smart" indentation to make it easy to write Elm code, especially
  * since Elm's parser depends on indentation as an implicit delimiter.
  */
-class ElmOnEnterSmartIndentHandler : EnterHandlerDelegateAdapter() {
+class ElmOnEnterIndentHandler : EnterHandlerDelegateAdapter() {
```

Comment:
A "smart enter handler" has an existing meaning, so I renamed this to avoid confusion. I can undo the change if you'd like.

Reply:
👍